reporting-cli/docs/dev/DEVELOPER_GUIDE.md

```diff
@@ -0,0 +1,36 @@
+## Developer Guide
+
+
+
+### SOP for installing and running reporting-cli
+
+1. Checkout this package from version control
+    ```
+    git clone git@github.com:opensearch-project/dashboards-reporting.git
+    cd dashboards-reports
+    git checkout main
+    ```
+2. Run `yarn` inside `dashboard-reports/reporting-cli/src`
```

Comment:
dashboard-reporting, and `yarn` usually runs in the same level as package.json (`dashboard-reports/reporting-cli`)

also there are two yarn.lock files, could you remove one? The correct one should be in the same dir as package.json

and i'm not sure if we need both yarn and npm since we then have to maintain both yarn.lock and package.json

Reply:
Deleted yarn files and added updated doc for using npm

Reply:
@joshuali925 It seems to me that we should maintain yarn-based install/run, since OSD and all plugins are yarn-based.

This would mean keep yarn.lock and updates to the README/ReferenceManual

Thoughts?

Reply:
for development or for using the cli? if we use yarn maybe don't keep the package-lock.json, my point is it doesn't make sense to keep both

Reply:
added yarn and removed package-lock.json